docs/how-it-works.mdx

```diff
@@ -0,0 +1,121 @@
+---
+sidebar_position: 2
+---
+
+# How it Works
+
+Learn the high-level workings of Divvi Up and how it accomplishes the
+privacy-preserving property.
+
+## Breakdown
+
+Divvi Up functionality can be broadly broken down into several steps.
+
+See the [Distributed Aggregation Protocol (DAP)][DAP] for a highly detailed
+specification of the protocol that underpins Divvi Up.
+
+<!-- TODO(inahga): Consider making this a more detailed `mermaid` diagram -->
+
+![](../static/how-it-works.png#gh-dark-mode-only)
+![](../static/how-it-works-light.jpg#gh-light-mode-only)
+
+### Step 0: Subscriber sets up Divvi Up
+
+Subscribers are entities, such as app developers, who wish to collect quantifiable metrics
+about their end-user applications.
+
+The subscriber chooses a Distributed Aggregation Protocol (DAP) client library, which is used for interacting with any DAP-compatible metrics aggregators, such as the ones provided by Divvi Up and its partners.
+A client library for [Rust (janus_client)][janus_client] and [TypeScript (divviup-ts)][divviup-ts] is available.
+
+For each metric, the subscriber creates a task, whose parameters encode data about the measurement being taken.
+
+### Step 1: Client generates measurements
+
+The subscriber writes and deploys app functionality that takes measurements corresponding to each task.
+Using a DAP client library, they also encode some of the task parameters into the app, such as the task identifier, measurement type, and the Divvi Up aggregators being used.
+
+For each measurement, the subsequent steps are performed.
+
+### Step 2: Client shards the measurement
+
+With the client library, the app randomly shards the measurement into two shares, known
+as report shares. One report share is for a leader aggregator, and the other for a
+helper aggregator.
+
+Both aggregators advertise a public key and possess its corresponding private key.
+The client library encrypts each share with the respective aggregator's public key.
+
+### Step 3: Client sends shares to the aggregators
+
+The client library sends each report share via HTTPS to the leader and helper
+aggregator.
+
+Aggregators reject out-of-range or invalid measurements, as defined in the task parameters. This is done without knowledge of the full input measurement.
+
+The aggregators in use are determined by the app developer and encoded in the
+task parameters. Each aggregator is operated independently by two different
+organizations. Often, Divvi Up will act as the leader, while another
+organization operates as a helper.
```

Comment:
```suggestion
The app developer determines which aggregators are used, and encodes their choices in the
task parameters. The two aggregator are operated independently by two different
organizations. Often, Divvi Up will act as the leader, while another
organization operates as a helper.
```

Reply:
I ended up striking this sentence, as it was redundant information--we established that the subscriber chooses the aggregators further up.